### PR TITLE
docs: copyedit dynamic imports

### PR DESCRIPTION
**Pull request by jbleyleSF**

### CODE REVIEW

File: text/0110-dynamic-components.md [hunk 1]
```diff
@@ -6,266 +6,277 @@ updated_at: 2021-02-01
...
+
 ## Motivation
 
-Before this RFC, LWC does not support dynamic imports nor allow you to specify at build or design time different component configurations to pivot on during runtime. We purposely made this decision to ensure we have a clean, statically analyzable and predictable behavior on which to build upon. Due to the complexity and dynamic nature of the Lightning Platform and OSS, as we expand LWC to new clouds, applications and contexts, this strategy isn't sufficient. New primitives must be introduced to guarantee and allow for a more flexible and scalable model.
+Before this RFC, LWC didn't support dynamic imports or allow you to specify at build or design time different component configurations to pivot on at run time. We purposely made this decision to ensure that we have clean, statically analyzable, and predictable behavior to build on. Due to the complexity and dynamic nature of the Lightning Platform and OSS, as we expand LWC to new clouds, applications, and contexts, this strategy isn't sufficient. New primitives must be introduced to guarantee and allow for a more flexible and scalable model.
```
Comment (ekashida) [inline text/0110-dynamic-components.md:59]: Long live the oxford comma!

File: text/0110-dynamic-components.md [hunk 2]
```diff
@@ -6,266 +6,277 @@ updated_at: 2021-02-01
...
-```json5
+```json
 {
-    "dynamicComponent": {
-        "loader": "@salesforce/loader",
-        "strict": true // only allow string identifiers
-    }
+  "dynamicComponent": {
+    "loader": "@salesforce/loader",
+    "strict": true // only allow string identifiers
```
Comment (ekashida) [inline text/0110-dynamic-components.md:253]: Nit: JSON doesn't allow for comments but JSON5 does

**Review by ekashida — APPROVED** [review]
This is great, thank you!

File: text/0110-dynamic-components.md [hunk 3]
```diff
@@ -6,266 +6,277 @@ updated_at: 2021-02-01
...
-You will notice that this follows the same pattern as what we have done for `@wire`: Bring the ability to understand and resolve dependencies at compile, build and design time.
+These invariants follow the same pattern that we use for `@wire`: the ability to understand and resolve dependencies at compile, build, and design time.
 
 ### API
 
-Below is the API and its ergonomics for dynamically loading a module. It has very similar mental model to the HTML `is` attribute proposal (which we can't use since Safari will not implement it in the foreseeable future): An arbitrary tag name on which you can assign any constructor.
+The API and ergonomics for dynamically loading a module has a similar mental model to the HTML `is` attribute proposal (which we can't use since Safari won't implement it in the foreseeable future): An arbitrary tag name on which you can assign any constructor.
 
-```json5
+```json
```
Comment (jbleyleSF) [inline text/0110-dynamic-components.md:249]:
```suggestion
```json5
```

File: text/0110-dynamic-components.md (hunk 2 above)
Comment (jbleyleSF) [reply text/0110-dynamic-components.md:253]: Thank you, Eugene. I thought that was a typo -- bad assumption. I looked up the syntax highlighting for json5, and iit seems to be `javascript`: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L2551

**Pull request merged**